test/internal/helpers/Player.java

```diff
@@ -1789,4 +1789,9 @@ public static Cleanups withValueOfAdventure(final int value) {
     cleanups.add(ConcoctionDatabase::refreshConcoctions);
     return cleanups;
   }
+
+  public static Cleanups withConcoctionRefresh() {
+    ConcoctionDatabase.refreshConcoctions();
+    return new Cleanups(new OrderedRunnable(ConcoctionDatabase::refreshConcoctions, 10));
```

Comment:
Magic number?  Not having finished everything I presume there is a desire to have something clean up last and 10 is an arbitrary value that will do that until there are > 10 cleanups?

Reply:
yeah, any number greater than one is fine.